src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Services/ResolveOverlappingTrainingDateRequestServiceTests.cs

```diff
@@ -164,11 +165,13 @@ private class ResolveOverlappingTrainingDateRequestServiceTestsFixture
             public OverlappingTrainingDateRequest OverlappingTrainingDateRequest { get; set; }
             private readonly ResolveOverlappingTrainingDateRequestService _sut;
             private readonly Mock<IOverlapCheckService> _overlapCheckService;
+            private UnitOfWorkContext UnitOfWorkContext { get; set; }
             public ProviderCommitmentsDbContext Db { get; set; }
 
             public ResolveOverlappingTrainingDateRequestServiceTestsFixture()
             {
                 _fixture = new Fixture();
+                UnitOfWorkContext = new UnitOfWorkContext();
```

Comment:
`UnitOfWorkContext` seems to have been added but not used?

Reply:
UnitOfWork instance is required for Publish